src/utils.rs

```diff
@@ -127,3 +117,47 @@ pub fn get_base_url() -> String {
         "https://api.soldeer.xyz".to_string()
     }
 }
+
+// Function to check for the presence of sensitive files or directories
+pub fn check_for_sensitive_files_or_directories(path: &Path) -> bool {
+    let env_file_exists = path.join(".env").exists();
+    let git_dir_exists = path.join(".git").exists();
```

Comment:
maybe we can just look for any `.file` we have? and just display them all? 
And let the user decide if it's safe?

Reply:
cause it may be more `.` files

Reply:
Makes sense yeah.